curriculum/challenges/_meta/lecture-working-with-css-grid/meta.json

```diff
@@ -0,0 +1,10 @@
+{
+  "name": "Working with CSS Grid",
```

Comment:
We added a few more videos in this section because the initial video wasn't covering enough.

We could add it to this PR or just create a separate PR since the comprehension check questions haven't been created yet.

Reply:
Done. The new challenge ID's are:
673226732b19aa1cacd0a75c
6732267ecab2151ced471cd4
6732268d05c3661d32a0fee8
6732269a7aa2ca1d6b6574fe
673226a62eb2121da41a3d68
673226afcd33991dd751937a
673226b97d4a731e0577ae93

I added them to the list on the project board.